app/routes/jobs/index.js

```diff
@@ -1,12 +1,20 @@
 import Route from '@ember/routing/route';
 import DataTableRouteMixin from 'ember-data-table/mixins/route';
-
 export default class JobsIndexRoute extends Route.extend(DataTableRouteMixin) {
   modelName = 'job';
 
+  queryParams = {
+    page: { refreshModel: true, replace: true },
+    size: { refreshModel: true },
+    sort: { refreshModel: true },
+    status: { status: this.shortStatus, refreshModel: true },
```

Comment:
@Windvis shortStatus is declared in the [job model](https://github.com/lblod/frontend-harvesting-self-service/blob/ebf9704d753c0a2e272887070bc27f432dc35718/app/models/job.js#L15), and it defines keywords for status values, i.e. `'http://redpencil.data.gift/id/concept/JobStatus/busy': 'busy',`

Reply:
@iurianu Yes, but there is no `shortStatus` property in the route so this will be undefined here. Furthermore, queryParams doesn't do anything with that status key. The only valid properties are `refreshModel`, `replace` and `as`: https://api.emberjs.com/ember/4.7/classes/Route/properties/queryParams?anchor=queryParams

Reply:
@Windvis you're right - I removed it, and it works just the same